### PR TITLE
DDPB-3705: Make date of birth and address fields mandatory

### PR DESCRIPTION
**Pull request by coderste**
## Description
As a OPG user
I want the COP users to enter the deputy DOB and I want the COP users to enter the deputy address
So that the deputy can be matched when the order is served and duplicates are not created in Sirius

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Merge check List

- [X] New functionality includes testing
- [ ] New functionality has been documented in the README if applicable **N/A**
- [ ] Approved by PMs

### CODE REVIEW

File: src/Form/DeputyForm.php [hunk 1]
```diff
@@ -51,7 +51,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('emailAddress', TextType::class, [
                 'label' => 'deputy.emailAddress.label',
-                'hint' => 'deputy.emailAddress.hint',
```
[the remark this answers is not in the document]
Comment (coderste) [reply src/Form/DeputyForm.php:54]: Yeah from what I understand Symfony doesn't even have a `hint` option for forms. It was causing UnitTests to fail as it was saying `hint` doesn't exist

[the remark this answers is not in the document]
Comment (Gugandeep) [reply src/Form/DeputyForm.php:54]: Looking into it you're right, I can't seem ti find a `hint `option.

Did find [this](https://symfony.com/blog/new-in-symfony-4-1-form-field-help) for a `help` option that's included in Symfony 4.1 should we want to use it in the future.

**Review by Gugandeep — DISMISSED** [review]
Looks good to me!

File: src/Entity/Deputy.php [hunk 2]
```diff
@@ -58,13 +60,15 @@ class Deputy
      * @var \DateTime
      *
      * @ORM\Column(name="dob", type="date", nullable=true)
+     * @Assert\NotBlank(message="deputy.dateOfBirth.notBlank")
```
Comment (acsauk) [inline src/Entity/Deputy.php:63]: For all of these where we refer to `message="<SOMETHING>"` you'll need to add the corresponding translation to `/translations/validators.en.yml`.

File: src/Form/DeputyForm.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (acsauk) [reply src/Form/DeputyForm.php:54]: You can remove the corresponding translation now as well.

**Review by acsauk — CHANGES_REQUESTED** [review]
Good job - just the translations to add for the validation messages and then good to go 👍

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**